src/main/resources/i18n/messages.properties

```diff
@@ -2150,4 +2150,12 @@ EditableParagraph.tooltip=Edit
 # ========================================================================================== #
 # ADMINISTRATIVE PANEL                                                                       #
 # ========================================================================================== #
-admin.panel.title=Admin Panel
\ No newline at end of file
+admin.panel.title=Admin Panel
+admin.panel.users=Users
```

Comment:
Try to name your keys after the component they are tied to e.g. `AdminPanel.title` or whatever the component is.  That way it is easy to track down where it is used.  We have had problems deleting keys that get re-used.  I have also begun using a system that if it is internationalized on the server (e.g. the response from a request), preface the key with `server.`  just makes it easier to track down.